src/app/donation-start/donation-start-form/donation-start-form-parent.component.ts

```diff
@@ -190,6 +190,7 @@ export class DonationStartFormParentComponent implements AfterContentChecked, Af
    * Keys are ISO2 codes, values are names.
    */
   public countryOptionsObject: Record<string, string>;
+  paymentMethodChecked: boolean;
```

Comment:
I think we haven't been sticking this for all the new stepper work, but noticing here since it's between a private and a public: probably worth specifying which one a property is generally instead of letting it be public by default.

Reply:
```suggestion
  private paymentMethodChecked: boolean;
```

Reply:
I'm also not clear if or why new should have an additional property. Shouldn't `!!this.selectedSavedMethod` always evaluate to the same thing as `this.paymentMethodChecked`?